python/sglang/srt/layers/sampler.py

```diff
@@ -35,6 +36,24 @@ def forward(
     ):
         logits = logits_output.next_token_logits
 
+        # Apply the custom logit processors if registered in the sampling info.
+        if sampling_info.custom_logit_processor is not None:
+            for (
```

Comment:
```suggestion
        # Apply the custom logit processors if registered in the sampling info.
        if sampling_info.custom_logit_processor is not None:
            self.apply_custom_logit_processor(logits, sampling_info. custom_logit_processor)
```

minor style comment: move this part into a separate function.